deploy/2-deploy-redstone-oracle.ts

```diff
@@ -36,5 +36,6 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
   }
 };
 
+func.skip = async ({ network }: HardhatRuntimeEnvironment) => network.name !== "arbitrumone";
```

Comment:
Wouldn't it be the other way? I assume we don't want to deploy the Redstone oracle if we are on `arbitrumone`

Reply:
I have changed it to skip deployment if network is **arbitrumone** for now https://github.com/VenusProtocol/oracle/pull/174/commits/5f4f8be3d496f4e7df1ad32a54a4aa6acf5ee422

Reply:
I think we want to deploy the RedStone oracle only to these specific networks: `hardhat`, `bscmainnet`, `bsctestnet`, `sepolia`, `ethereum`. Otherwise, I would skip the deployment

Reply:
```
func.skip = async ({ network }: HardhatRuntimeEnvironment) => !["hardhat", "bscmainnet", "bsctestnet", "sepolia", "ethereum"].includes(network.name);
```

Reply:
Done https://github.com/VenusProtocol/oracle/pull/174/commits/d8fdd931b61d1e4ffe5cd89d7abbd1ddc059d0aa